ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/SentMessage.qml

```diff
@@ -4,7 +4,6 @@ import "../../../../../imports"
 
 StyledText {
     id: sentMessage
-    visible: isCurrentUser && !timeout && !isExpired && isMessage
```

Comment:
Removing this might affect the CompactMessage. Have you checked what it looks like in compact mode?

Reply:
So I checked compact messages and it looks indeed broken, however it already looks broken on latest master. These changes don't actually make it worse, rather better.

I've updated the `SentMessage` component in the compact message to behave the same as in the normal message.

However, we should revisit the compact UI.